apps/docs/generator/src/main.ts

```diff
@@ -38,4 +45,28 @@ function main(): void {
   }
 }
 
+function generateConstraintTypeDocs(rootPath: string): void {
+  const docsPath = join(
+    rootPath,
+    'apps',
+    'docs',
+    'docs',
+    'user',
+    'constraint-types',
+  );
+  registerConstraints();
```

Comment:
Somehow constraints are not registered by default?

Reply:
They are registered when calling `createJayveeServices` but that needs to be refactored in the future 😅

So currently it's fine to call `registerConstraints` manually if there is no need to call `createJayveeServices`.